std/hash/cityhash.zig

```diff
@@ -214,7 +214,7 @@ pub const CityHash64 = struct {
     }
 
     fn hashLen0To16(str: []const u8) u64 {
-        const len: u64 = @truncate(u64, str.len);
+        const len: u64 = u64(str.len);
```

Comment:
I have no problem with this, but just so you know, it's redundant with a typed variable:

```zig
         const len = u64(str.len); 
```
exactly equivalent to

```zig
         const len: u64 = str.len;
```